CONTRIBUTING.md

```diff
@@ -49,15 +49,15 @@ The documentation is using [docsify](https://docsify.js.org/) to render.
 
 #### Adding examples
 
-- Run `python docs/generate_examples.py`. This will get all examples from `docs/examples/cases`, render the resulting HTML and
+- Run `poetry run python docs/generate_examples.py`. This will get all examples from `docs/examples/cases`, render the resulting HTML and
  include it in the appropriate folders.
 - (optional) If you have added an example, add the file name (without `.json`), the display name and description in `docs/cases_description.yaml`
 
 #### Generating locally
 
 Execute following script from root dir of the repository
 ```bash
-python3 docs/generate_examples.py
+poetry run python docs/generate_examples.py
```

Comment:
I've tried running this command and I now have a bunch of dirty files (according to git) in `docs/examples/...`. From the looks it it, it seems like my local files are "formatted" while the ones in the repo have had all their whitespace stripped out.

I'm not sure if it's related to your change here. Maybe I just stumbled on a different problem.

Reply:
It's probably because of https://github.com/coveooss/json-schema-for-humans/commit/9fd2752dddb29fd2dfcf5e6a1968569619bf7d74, I'll fix